hack/vagrant/Vagrantfile

```diff
@@ -0,0 +1,16 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.boot_timeout=600
+  config.vm.box = "bento/ubuntu-16.04"
+  config.vm.provision :shell, path: "bootstrap.sh"
+  config.vm.synced_folder "../../", "/go/src/github.com/alibaba/pouch"
+  
+  config.vm.define "node0" do |node0|
```

Comment:
`node0` looks like random composed name. Can you change it to the `pouch-dev-node` something like that?